tv.kodi.Kodi.json

```diff
@@ -0,0 +1,128 @@
+{
+    "app-id": "tv.kodi.Kodi",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk9"
+    ],
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=home",
+        "--share=ipc",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--socket=wayland",
+        "--socket=x11",
+        "--system-talk-name=org.bluez",
+        "--system-talk-name=org.freedesktop.Avahi"
+    ],
+    "command": "kodi",
+    "post-install": [
+        "sed -i 's|^KODI_DATA.*|KODI_DATA=${HOME}/.var/app/tv.kodi.Kodi/data|' /app/bin/kodi"
+    ],
+    "modules": [
+        "deps/avahi.json",
+        "deps/bluez.json",
+        "deps/fribidi.json",
+        "deps/fstrcmp.json",
+        "deps/ghostscript.json",
+        "deps/glu.json",
+        "deps/libass.json",
+        "deps/libcdio.json",
+        "deps/lzo2.json",
+        "deps/samba.json",
+        "deps/swig.json",
+        "deps/taglib.json",
+        "deps/tinyxml.json",
+        "deps/udev.json",
+        {
+            "name": "kodi",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DVERBOSE=1",
+                "-DENABLE_INTERNAL_FFMPEG=ON",
+                "-DENABLE_INTERNAL_FLATBUFFERS=ON",
+                "-DENABLE_INTERNAL_FMT=ON",
+                "-DENABLE_INTERNAL_RapidJSON=ON",
+                "-DFLATBUFFERS_URL=build/download/flatbuffers-1.9.0.tar.gz",
+                "-DCROSSGUID_URL=build/download/crossguid-8f399e8bd4.tar.gz",
+                "-DFMT_URL=build/download/fmt-5.1.0.tar.gz",
+                "-DRapidJSON_URL=build/download/rapidjson-1.1.0.tar.gz",
+                "-DFFMPEG_URL=build/download/4.0.2-Leia-Alpha3.tar.gz",
+                "-DLIBDVDCSS_URL=build/download/1.4.1-Leia-Beta-3.tar.gz",
+                "-DLIBDVDREAD_URL=build/download/6.0.0-Leia-Alpha-3.tar.gz",
+                "-DLIBDVDNAV_URL=build/download/libdvdnav-6.0.0-Leia-Alpha-3.tar.gz",
+                "-DJava_JAVA_EXECUTABLE=/usr/lib/sdk/openjdk9/bin/java"
+            ],
+            "post-install": [
+                "mv /app/share/applications/kodi.desktop /app/share/applications/tv.kodi.Kodi.desktop"
```

Comment:
Using `install` is more suited here

Reply:
Using `rename-desktop-file` is even more suited here :)
and `rename-icon` is also necessary

Reply:
found it